src/app/(proper_react)/Shell.module.scss

```diff
@@ -3,12 +3,17 @@
 .wrapper {
   height: 100%;
   display: flex;
+  gap: 0;
+
+  @media screen and (min-width: $screen-lg) {
+    gap: $layout-md;
+  } 
 }
 
 .mainMenu {
   display: none;
 
-  @media screen and (min-width: $screen-md) {
+  @media screen and (min-width: $screen-lg) {
```

Comment:
Hmm, with just this change, the sidebar menu already disappears when the viewport width is smaller than `$screen-lg`, but the mobile menu only shows up when it's smaller than `$screen-md` - so in between, there's no way to navigate the site. I'm not sure why this change was made, but either we'll need an equivalent change in `MobileShell.module.scss`, or to revert this one.

(I'd lean towards the latter, as the mobile toolbar looks rather weird on screens as wide as `$screen-lg` - feels like there should be plenty of room for a sidebar. Instead, we probably want the Exposure Cards to switch to their thin layouts more quickly - either using [Container Queries](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Container_Queries) (not supported in older Safari) or using `flex-wrap`.)

Reply:
```suggestion
  @media screen and (min-width: $screen-md) {
```

Otherwise there's no navigation on some screen sizes, and that breakage is easier to miss than the exposure cards not fitting:

![image](https://github.com/mozilla/blurts-server/assets/4251/ab00b1e0-3d6d-48cc-b70c-6cf284bf6ce7)